ui/src/app/kvm/components/KVMActionFormWrapper/ComposeForm/ComposeFormFields/ComposeFormFields.tsx

```diff
@@ -10,27 +10,43 @@ import FormikField from "app/base/components/FormikField";
 import ResourcePoolSelect from "app/base/components/ResourcePoolSelect";
 import ZoneSelect from "app/base/components/ZoneSelect";
 import type { Pod } from "app/store/pod/types";
+import { PodType } from "app/store/pod/types";
 import { getRanges } from "app/utils";
 
 type Props = {
   architectures: Pod["architectures"];
   available: {
     cores: number;
+    hugepages: number;
     memory: number; // MiB
     pinnedCores: number[];
   };
   defaults: ComposeFormDefaults;
+  podType: PodType;
+};
+
+const getHugepagesTooltip = (isLxd: boolean, hasFreeHugepages: boolean) => {
+  if (!isLxd) {
+    return "Hugepages are only supported on LXD KVMs.";
```

Comment:
We often mix the use of KVM and VM. In this case does LXD refer to them as KVMs?

Reply:
Yeah this refers to them as KVMs.